bin/cpanorg_perl_releases

```diff
@@ -122,14 +122,18 @@ sub fetch_perl_version_data {
     my @perls;
     my @testing;
     foreach my $module ( @{ $data->{releases} } ) {
-        next unless $module->{authorized} eq 'true';
+        next unless $module->{authorized};
 
         my $version = $module->{version};
 
         $version =~ s/-(?:RC|TRIAL)\d+$//;
         $module->{version_number} = $version;
 
-        my ( $major, $minor, $iota ) = split( '[\._]', $version );
+        my ( $major, $minor, $iota ) = split( /[\._]/, $version );
+        if ( !defined $iota ) {
+            $minor .= '0'x(6 - length $minor) if length $minor < 6;
```

Comment:
Would sprintf be cleaner here?

Reply:
I don't think sprintf can be used this way - the decimal component is already parsed out at this point and we want to right-pad zeroes, not left.

Reply:
I missed the right padding bit.